kratos/solving_strategies/builder_and_solvers/residualbased_block_builder_and_solver_with_lagrange_multiplier.h

```diff
@@ -281,10 +281,9 @@ class ResidualBasedBlockBuilderAndSolverWithLagrangeMultiplier
         BaseType::BuildAndSolve(pScheme, rModelPart, rA, rDx, rb);
 
         // Update the Lagrange multiplier solution
-        #pragma omp parallel for
-        for (int i = 0; i < static_cast<int>(BaseType::mSlaveIds.size()); ++i) {
-            mLagrangeMultiplierVector[i] += rDx[BaseType::mEquationSystemSize + i];
-        }
+        IndexPartition<unsigned int>(static_cast<int>(BaseType::mSlaveIds.size())).for_each([&](unsigned int Index){
```

Comment:
size_t

Reply:
Actually I used this before, but it was showing some error , related to this line (on 284)`BaseType::mEquationSystemSize + Index`, addition is not possible ...If I am not wrong. I will try again and will let you know

Reply:
@philbucher, this was the error (in CI)
![image](https://user-images.githubusercontent.com/46954821/119816342-371a9800-bf0a-11eb-9759-5decf571ddc3.png)